src/DefaultRules.php

```diff
@@ -79,6 +79,11 @@ abstract class DefaultRules
                 'method',
             ],
         ],
+        'yoda_style'                => [
```

Comment:
We have this in line 41:
```
'yoda_style'                             => false,
```
Does this not already change to non-yoda-style?

Reply:
According to https://mlocati.github.io/php-cs-fixer-configurator/#version:3.5|fixer:yoda_style this is a different setting 🙁

If I get it right, `'yoda_style' => false` will just disable the rule and not setting it to false.

According to my test the following will result in:
```php
$test = null;

if (1 === $test) {
}

if ('1' === $test) {
}

if (null === $test){
}

echo null !== $test ? 'notNull' : 'null';
```
config: `    'yoda_style' => false,`
```php
$test = null;

if (1 === $test) {
}

if ('1' === $test) {
}

if (null === $test) {
}

echo null !== $test ? 'notNull' : 'null';
```
config: `    'yoda_style' => true,`
```php
$test = null;

if (1 === $test) {
}

if ('1' === $test) {
}

if (null === $test) {
}

echo null !== $test ? 'notNull' : 'null';
```
config: `    'yoda_style' => [ 'equal'            => false, 'identical'        => false, 'less_and_greater' => false, ],`
```php
$test = null;

if ($test === 1) {
}

if ($test === '1') {
}

if ($test === null) {
}

echo $test !== null ? 'notNull' : 'null';
```

Reply:
Thanks for verifying! 🙂 I have removed the other yoda_style configuration to prevent conflicts. It was not a valid value, anyway, as per the documentation 😅